src/js/shield.js

```diff
@@ -382,8 +399,15 @@ export function generateShieldCtx(map, id) {
   // Convert numbering systems. Normally alternative numbering systems should be
   // tagged directly in ref=*, but some shields use different numbering systems
   // for aesthetic reasons only.
-  if (routeDef.ref && shieldDef.numberingSystem === "roman") {
-    routeDef.ref = romanizeRef(routeDef.ref);
+  if (routeDef.ref) {
+    switch (shieldDef.numberingSystem) {
+      case "arab":
+        routeDef.ref = latinToArabicDigits(routeDef.ref);
```

Comment:
> Roads in Saudi Arabia are signed solely in Eastern Arabic numerals. This PR contains code to convert digits to Eastern Arabic.

In #676, I included a Roman numeral conversion function because the Roman numerals were purely decorative. In plain text (and even some signs), the route numbers are in Western Arabic numerals. By contrast, Eastern Arabic is the local language’s numeral system. Would it be more appropriate to render the number however it’s tagged in `ref`, allowing mappers the flexibility to shunt the transliterated number to a subkey such as `ref:Latn`?

/ref https://github.com/ZeLonewolf/openstreetmap-americana/issues/467#issuecomment-1169234746

Reply:
- This is a philosophical question ineeded. Off the cuff I would say the ref should have the Eastern Arabic numbers in them, which we would render with the right font.  This PR takes a more resilient approach and papers over the difference at the expense of not exposing what we might agree is an error.

Reply:
It would be worth asking the folks that implemented the scheme though I'm not sure who to ask.

Reply:
Note that most of Saudi Arabia’s highways are also part of the Arab Mashreq International Road Network, which is signposted in Western Arabic numerals: #466.